### DISCUSSION TITLE
Possibility to adjust height

### DISCUSSION THREAD

**Discussion by kristianberge** [702573N/Obsidian-Tasks-Calendar#36] [Ideas]
Hello,

Congrats on tasks calendar! Really enjoy it! Is it however possible to adjust the height of the calendar? Specifically I would love to be able to use week view style 4 as part of my work dashboard but than as a horizontal strip with only the current week. Could you give me a tip how I could change the js or css to for example let the style 4 only be 300 px in height? Or would it be a possibility to implement this as one of the options?

**Comment by 702573N** [comment]
You can add a custom style to style4 class:
```
.tasksCalendar[view='week'].style4 .grid { height: 300px !important; }
```

**Reply by kristianberge** [reply]
fantastic! thank you!

**Reply by kristianberge** [reply]
is it also possible to wrap the task text? in style4 I only see the first few words for each task.

**Comment by 702573N** [comment]
You can enable lineClamp inside the calendar by using the options lineClamp1, lineClamp2, lineClamp3, or get full task description with the option noLineClamp.

https://github.com/702573N/Obsidian-Tasks-Calendar#options
________________________________
Von: kristianberge ***@***.***>
Gesendet: Saturday, December 10, 2022 1:19:57 PM
An: 702573N/Obsidian-Tasks-Calendar ***@***.***>
Cc: 702573N M4275CH1NK3 ***@***.***>; Comment ***@***.***>
Betreff: Re: [702573N/Obsidian-Tasks-Calendar] Possibility to adjust height (Discussion #36)


is it also possible to wrap the task text? in style4 I only see the first few words for each task.

**Reply by kristianberge** [reply]
sorry. thanks!

**Reply by kristianberge** [reply]
last question. In week style4, is it somehow possible to hide saturday and sunday?

**Comment by 702573N** [comment]
This is more complex. Add this three css statements to view.css file:

```
.tasksCalendar[view='week'].style4 .cell[data-weekday="0"] { display: none !important }

.tasksCalendar[view='week'].style4 .cell[data-weekday="6"] { display: none !important }

.tasksCalendar[view='week'].style4 .grid, .iconStyle4 { grid-template-rows: 1fr; grid-template-columns: 1fr 1fr 1fr 1fr 1fr; }

```

**Reply by kristianberge** [reply]
hmmm. while this do hide saturday and sunday it does not allow the remaining days (mon-fri) to use the space. Looks like this:
<img width="637" alt="CleanShot 2022-12-10 at 22 03 12@2x" src="https://user-images.githubusercontent.com/118932013/206875228-8d21a7a7-00f4-4b37-9c56-413520026bac.png">

**Reply by kristianberge** [reply]
hi again, strange but after a restart of obsidian it suddenly worked!! thank you so much for the very fast solutions!!!
<img width="902" alt="CleanShot 2022-12-10 at 22 11 59@2x" src="https://user-images.githubusercontent.com/118932013/206875448-5ba37463-3220-49da-908f-84ab069bb881.png">

**Comment by 702573N** [comment]
Please check the latest release [1.3.4](https://github.com/702573N/Obsidian-Tasks-Calendar/releases/tag/1.3.4)
